<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StandardDialogs</class>
 <widget class="QWidget" name="StandardDialogs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StandardDialogs</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>741</width>
     <height>451</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>输入对话框 </string>
    </attribute>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>19</y>
       <width>711</width>
       <height>391</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="btn_getint">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string> getInt()</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_getdouble">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>getDouble()</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_getitem">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>120</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>getItem</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_gettext">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>160</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>getText()</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_getmtext">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>200</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>getMultiText()</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_1">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>40</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>int</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>80</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>double</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>120</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>item</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>160</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>text</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>200</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>multitext</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>颜色对话框 </string>
    </attribute>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>621</width>
       <height>341</height>
      </rect>
     </property>
     <widget class="QPushButton" name="btn_getcolor">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>150</width>
        <height>100</height>
       </rect>
      </property>
      <property name="text">
       <string>getColor</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_color">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>140</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>是否显示透明度</string>
      </property>
     </widget>
     <widget class="QToolButton" name="check_alpha">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>字体对话框 </string>
    </attribute>
    <widget class="QPushButton" name="btn_font">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>getFont()</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_font">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>120</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string> 你好,Qt</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>文件对话框 </string>
    </attribute>
    <widget class="QLineEdit" name="line_path">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>150</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_dir">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>60</y>
       <width>200</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>getExistingDirectory()</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_file">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>100</y>
       <width>200</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>getOpenFileName()</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_files">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>140</y>
       <width>200</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>getOpenFileNames()</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_savefile">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>180</y>
       <width>200</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>getSaveFilenames()</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>消息对话框 </string>
    </attribute>
    <widget class="QPushButton" name="btn_critical">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>critical()</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_info">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>information()</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_question">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>180</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>question()</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_warning">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>60</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>warning()</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_message">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>120</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>showMessage()</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>进度对话框 </string>
    </attribute>
    <widget class="QPushButton" name="btn_progress">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>110</y>
       <width>75</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>  进度</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_7">
    <attribute name="title">
     <string>向导对话框 </string>
    </attribute>
    <widget class="QPushButton" name="btn_wizard">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>150</y>
       <width>75</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string> 弹出向导</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
